src/libraries/System.Runtime.Loader/ref/System.Runtime.Loader.cs

```diff
@@ -10,6 +10,7 @@ public static partial class AssemblyExtensions
     {
         [System.CLSCompliantAttribute(false)]
         public unsafe static bool TryGetRawMetadata(this System.Reflection.Assembly assembly, out byte* blob, out int length) { throw null; }
+        public static void ApplyUpdate(Assembly assembly, ReadOnlySpan<byte> metadataDelta, ReadOnlySpan<byte> ilDelta, ReadOnlySpan<byte> pdbDelta = default) { throw null; }
```

Comment:
I'm surprised we decided to make the last parameter optional given that we don't actually expect many folks to be calling this method.

Reply:
I'll remove the default on the pdbDelta parameter.